log4j-core/src/main/java/org/apache/logging/log4j/core/config/json/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Classes and interfaces supporting configuration of Log4j 2 with JSON.
  */
 @Export
-@Version("2.20.1")
+@Version("2.21.0")
```

Comment:
```suggestion
@Version("2.26.0")
```

See [Fixing API compatibility check failures](https://github.com/apache/logging-log4j2/blob/2.x/BUILDING.adoc#fixing-api-compatibility-check-failures).